README.md

```diff
@@ -123,16 +123,24 @@
 #### 🏷️ Tags
 
 - #### `unmaintained`
-  **(boolean|string)** - signify that a library is no longer maintained. Can also pass the package name string to point out the recommended alternative.
+  **(boolean)** - signify that a library is no longer maintained. Provide alternative libraries with the `alternatives` field.
```

Comment:
```suggestion
  **(boolean)** - signify that a library is no longer maintained. You can provide alternative or replacement libraries with the `alternatives` field, if needed.
```

Let's make it clear, that those additional fields are not required, when you set a flag.